source/includes/_eu.md

```diff
@@ -0,0 +1,208 @@
+# EU PSD2 API
+
+<aside class="notice">
+    Monzo's EU PSD2 Dedicated Interface API is currently only available for testing in the sandbox environment. Production API access will be available at a later date. 
```

Comment:
also probably worth mentioning that TPPs will need to contact us to [gain access ](https://github.com/monzo/wearedev/blob/76dbafed0f8a76b701c022ac86bc7259bfc8f54f/edge-proxy.open-banking/openbankingfilter/filter.go#L62-L67)initially